src/Datadog.Trace.ClrProfiler.Managed/Integrations/HttpMessageHandlerIntegration.cs

```diff
@@ -187,7 +187,7 @@ public static class HttpMessageHandlerIntegration
                        .Start(moduleVersionPtr, mdToken, opCode, Send)
                        .WithConcreteType(httpMessageHandler)
                        .WithParameters(request, cancellationToken)
-                       .WithNamespaceAndNameFilters(NamespaceAndNameFilters)
+                       .WithNamespaceAndNameFilters(ClrNames.HttpResponseMessage, ClrNames.HttpRequestMessage, ClrNames.CancellationToken)
```

Comment:
Doesn't this return a `Task<HttpResponseMessage>`?

Reply:
No, this is the new synchronous Send API in .NET 5 😄 I specifically had to change this because `NamespaceAndNameFilters` was incorrectly setting a return type of `Task<HttpResponseMessage>`

Reply:
My bad. Something in the github UI maybe we thing this was inside `HttpMessageHandler_SendAsync()`. Thanks!